client/knode/genesis/generator.go

```diff
@@ -85,23 +98,27 @@ func (gen *generator) Generate(opts Options) (*core.Genesis, error) {
 	return genesis, nil
 }
 
+//genesisAllocFromOptions changes alloc property based on valid Genesis Options.
 func (gen *generator) genesisAllocFromOptions(opts *validGenesisOptions) error {
 	if err := gen.deployContracts(opts); err != nil {
 		return err
 	}
 
 	gen.prefundAccounts(opts.prefundedAccounts)
-	gen.addBatchOfPrefundedAccountsIntoGenesis()
+	// @TODO (rgeraldes) - is this necessary?
```

Comment:
I mean the functional side. There is a reason why that prefund was there in the first place.

Reply:
Sorry I cant understand what you mean, this pre fetch was there because the original cmd/genesis added this batch accounts. The reason is unknown to me. I removed when I saw your comment thinking that you meant if this was necessary and I could not see any difference so I removed. 

What should I do?